front/src/applications/stdcm/utils/formatSimulationReportSheet.ts

```diff
@@ -142,3 +142,41 @@ export function getOperationalPointsWithTimes(
 
   return opResults;
 }
+
+/**
+ * Converts a base64 image into a JPEG blob while reducing quality.
```

Comment:
Do we accept this low quality or it was already like that before ? (open the img in full screen)

![Capture d’écran 2024-10-04 à 11 19 54](https://github.com/user-attachments/assets/3e2d2cab-4e53-4f7e-ad24-b9d79015e44e)

Reply:
Yes it used to be like that, it depends on the resolution of your screen, here's a screen I just took with 80% quality : 

<img width="687" alt="Capture d’écran 2024-10-04 à 11 43 03" src="https://github.com/user-attachments/assets/cd627c50-1ffa-4d5d-aee1-0b1720ffe0a9">

Reply:
Anyways, once we have information on the maximum authorized weight, we'll be able to adapt the degree of quality reduction.